### PR TITLE
Ban some plugins from the plugin listing

### PR DESCRIPTION
**Pull request by keithamus**
This bans 2 plugins. Anyone from @chaijs/chai-docs care to take a look?

### CODE REVIEW

File: _scripts/banned_plugins.js [hunk 1]
```diff
@@ -2,5 +2,7 @@ module.exports = [
 
   'chai-51st-state', // Banned for profane language
   'chai2.0-webdriver-promised', // Fork with poor metadata
+  'lodash-match-pattern', // not a real chai plugin, chai-match-pattern is the real one
```
Comment (astorije) [inline _scripts/banned_plugins.js]: Opened https://github.com/Originate/lodash-match-pattern/pull/7 for this. Let's see if they remove quickly so we can avoid this.

**Comment by astorije** [comment]
(By the way, @keithamus, is the LGTM thing fixed, aka, can other members than you merge now?)

File: _scripts/banned_plugins.js [hunk 2]
```diff
@@ -2,5 +2,7 @@ module.exports = [
...
+  'shimo-chai-as-promised', // dupe of chai-as-promised
```
Comment (astorije) [inline _scripts/banned_plugins.js]: Opened https://github.com/shimohq/chai-as-promised/pull/1 for this. Same as above.

**Comment by lucasfcosta** [comment]
@astorije We both can merge this, we just need to have enough LGTMs before doing so.
LGTM too.

**Comment by astorije** [comment]
I wasn't 👍 on this btw, I think it's better to try to fix said repos instead of maintaining a list of banned ones.

**Comment by lucasfcosta** [comment]
@astorije Ah, I'm sorry, I thought that as you've asked here if @keithamus was the only one who could merge this you were agreeing to merge this now until the issues you opened got fixed. I'm sorry for this, it was totally my fault. Ah, and just as tip for other people reading this: let's avoid using the keyword LGTM without explicitly meaning that a PR is approved, otherwise LGTM.co thinks it is an approval and it counts towards the minimum approval necessary to merge something.

Sorry again @astorije, I really thought you were 👍 on this.

**Comment by astorije** [comment]
> Sorry again @astorije, I really thought you were 👍 on this.

No worries at all, it's no big deal, I'll open a PR removing these when the repos will merge/publish their changes ;-)

> Ah, and just as tip for other people reading this: let's avoid using the keyword LGTM without explicitly meaning that a PR is approved, otherwise LGTM.co thinks it is an approval and it counts towards the minimum approval necessary to merge something.

Ah ah, I had no idea. I have never used that service before, it kind of triggers my OCD to see these orange/pending checks in all non-approved PRs :-P

**Comment by keithamus** [comment]
Hey, no big deal, we can remove them from the blacklist as and when the PRs @astorije submitted get merged. I'd personally prefer to see us do this to quickly remove the problem and remove confusion for users, and then address the problem properly. My manager would describe this as "tactical then strategic" (but I shudder at that).

**Comment by astorije** [comment]
Fully agree with you, @keithamus (including the shudder 😄). OK to ban things ASAP and try to have them clean up on their end too.

**Pull request merged**